lib/Auth/ApiAuth.php

```diff
@@ -11,11 +11,23 @@
 
 namespace Mautic\Auth;
 
+use Psr\Http\Client\ClientInterface;
+
 /**
  * OAuth Client modified from https://code.google.com/p/simple-php-oauth/.
  */
 class ApiAuth
 {
+    /**
+     * @var \Psr\Http\Client\ClientInterface|null
+     */
+    protected $client;
```

Comment:
Since we've dropped the PHP 7.4 support then we can start using typed properties

```php
protected ClientInterface $client
```

Reply:
I've added types to all properties I've touched.